programming basics/Conda and Git.ipynb

```diff
@@ -0,0 +1,134 @@
+{
```

Comment:
<ul><li>For the <code>git commit</code> line, I think it would be better to introduce them to the long hand version of <code>git commit</code> , where you enter the message in a window. Probably good to add a link to how to write good commit message.</li><li>the last <code>git push origin</code> has extra quotation mark.</li><li>Do we also want to introduce <code>git branch</code>?  <span class="ql-cursor">﻿</span></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PTC-CMC/cybercamp/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>